src/v3/pages/reports/meeting_attendance/components/report_results/index.styles.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Divider } from '@mui/material';
+import { styled } from '@mui/system';
+
+export const StyledContentBox = styled(Box)({
+  gap: '16px',
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+export const StyledContentCardBox = styled(Box)({
+  backgroundColor: 'var(--white)',
+  border: '1px solid var(--accent-300)',
+  borderRadius: 'var(--radius-xl)',
+  padding: '16px',
+  display: 'flex',
+  gap: '16px',
+  flexDirection: 'column',
+  flex: 19,
+});
+
+export const StyledColumnBox = styled(Box)(({ gap }: { gap: string }) => ({
```

Comment:
@Logan08691: please make sure this `gap` prop does not create the invalid prop on a DOM element error.